### DISCUSSION TITLE
Hover, Focus, and Other States doesn't work?

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#2943] [Q&A] [answered]
```py
ui.button("TEST").classes("hover:bg-red")
```
All these state change seems don't work

**Comment by rodja** [comment]
This comes up regularly: for styling buttons you should stick to the Quasar props (see #1920 for an explanation). We improved colorization of buttons in #371 and #1821 but tailwind classes are just not the right way to do this. But you can create your own `ui.element('button')` without any styling and then beef it up with the required classes.

**Comment by falkoschindler** [comment, marked as answer]
As a workaround you can use some custom CSS:

```py
ui.add_css('button.my-red-button:hover { background-color: red !important; }')

ui.button('TEST').classes('my-red-button')
```
